src/dlgAboutDialog.cpp

```diff
@@ -1103,17 +1106,157 @@ void dlgAboutDialog::setSupportersTab(const QString& htmlHead)
 
 QString dlgAboutDialog::createBuildInfo() const
 {
+#if defined(Q_OS_WIN32)
+    // The build environment is for Windows one - which could be run
+    // native on a 32-bit or 64-bit CPU or inside the WOW64 sub-system on a
+    // 64-bit one:
+
+    auto hProcess = GetCurrentProcess();
+    BOOL value = false;
+    std::optional<bool> isWow64Process;
+    auto result = IsWow64Process(hProcess, &value); // hProcess is a "HANDLE"
+    if (!result) {
+        // Failed to work - so there is no value to assign to isWow64Process:
+        qWarning().nospace().noquote() << "dlgAboutDialog::createBuildInfo() WARNING - IsWow64Process(...) failed, WOW64 status unknown.";
+    } else {
+        isWow64Process = static_cast<bool>(value);
+    }
+    if (Q_UNLIKELY(QLatin1String(qVersion()) != QLatin1String(QT_VERSION_STR))) {
+        // Originally QLatin1String::compare(...) was to be used but, despite
+        // the Qt documentation claiming that that method was introduced in
+        // Qt 5.14 it lies; CI builds using Qt version 5.14.2 were borking
+        // and giving an: "error: ‘class QLatin1String’ has no member named
+        // ‘compare’" message instead!
+        return qsl("<table border=\"0\" style=\"margin-bottom:18px; margin-left:36px; margin-right:36px;\" width=\"100%\" cellspacing=\"2\" cellpadding=\"0\">\n"
+                   "<tr><td colspan=\"2\" style=\"font-weight: 800\">%1</td></tr>\n"
+                   "<tr><td style=\"padding-right: 10px;\">%2<td>%3</td></tr>\n"
+                   "<tr><td style=\"padding-right: 10px;\">%4</td><td>%5</td></tr>\n"
+                   "<tr><td style=\"padding-right: 10px;\">%6</td><td>%7</td></tr>\n"
+                   "<tr><td style=\"padding-right: 10px;\">%8</td><td>%9</td></tr>\n"
+                   "<tr><td style=\"padding-right: 10px;\">%10</td><td>%11</td></tr>\n"
+                   "</table>")
+                .arg(tr("Technical information:"), // %1
+                     tr("Version"), // %2
+                     mudlet::self()->scmVersion, // %3
+                     tr("OS"), // %4
+                     QSysInfo::prettyProductName(), // %5
+                     /*: This is shown for 32-Bit Windows builds when run on a
+                      *64-Bit OS. \"WoW64\" stands for WindowOnWindows64.
+                      */
+                     isWow64Process.has_value() ? (isWow64Process.value() ? tr("CPU (WoW64)")
```

Comment:
Very few people know what WoW64 is, this is quite technical - how about this:

```suggestion
                     isWow64Process.has_value() ? (isWow64Process.value() ? tr("CPU (32bit on a 64bit machine)")
```

Reply:
WoW64 is the contracted M$ name for the process and the Wikipedia article https://en.wikipedia.org/wiki/WoW64 is not a bad reference for those wanting to know more. As it is a technical term it is also not likely to need have to be localised - the `tr(...)` is more for those locales who use something other than `CPU`!

Reply:
Most folks aren't interested in learning this for better or worse, they are more interested in playing text games, and that's okay.

Reply:
I've updated this PR to also put a message (more informative for the "release" version) - so that it'll still show WoW64 in that case but show the "bitness" otherwise.